src/java.desktop/unix/native/libawt/awt/awt_LoadLibrary.c

```diff
@@ -62,15 +62,24 @@ JNIEXPORT jboolean JNICALL AWTIsHeadless() {
         graphicsEnvClass = (*env)->FindClass(env,
                                              "java/awt/GraphicsEnvironment");
         if (graphicsEnvClass == NULL) {
+            // Not finding the class is not necessarily an error.
```

Comment:
I can't see how that comment can possibly be true. This change makes no sense.

Reply:
See [comment](https://github.com/openjdk/jdk/pull/20169#issuecomment-2247823182), please.